tpchgen/tests/integration_tests.rs

```diff
@@ -82,18 +76,7 @@ fn test_part_sf_0_001() {
     let sf = 0.001;
     let generator = PartGenerator::new(sf, 1, 1);
     test_generator(generator.iter(), "data/sf-0.001/part.tbl.gz", |part| {
-        format!(
```

Comment:
these formatting strings duplicate what is in the Display implementation, so I changed it to use the Display impl